internal/usage/usage.go

```diff
@@ -254,6 +254,7 @@ const (
 	SkipSampleData                  = "Don't load sample data into your Atlas Cluster."
 	ContainerRegion                 = "Atlas region where the container resides."
 	ContainerRegions                = "List of Atlas regions where the container resides."
+	ReclaimFreeSpaceTimestamp       = "The service reclaims the space after the current time passes the provided timestamp. If not set the current timestamp will be used."
```

Comment:
```suggestion
	ReclaimFreeSpaceTimestamp       = "The time at which the service reclaims the space. If not set, defaults to the current timestamp."
```

Reply:
Should we say what format the timestamp needs to be? "Timestamp in XXX format when the service reclaims the space." ?

Reply:
added